source/scripting_v3/GTA/Entities/Vehicles/Vehicle.cs

```diff
@@ -111,6 +135,12 @@ public void ToggleExtra(int extra, bool toggle)
 			Function.Call(Hash.SET_VEHICLE_EXTRA, Handle, extra, !toggle);
 		}
 
+		/// <summary>
+		/// Overrides this <see cref="Vehicle"/>'s engine audio with that of another vehicle.
+		/// </summary>
+		/// <param name="vehicleModelName">The internal name of the <see cref="Vehicle"/> to source engine sounds from, e.g. "DELUXO".</param>
+		public void OverrideEngineAudio(string vehicleModelName) => Function.Call(Hash.FORCE_USE_AUDIO_GAME_OBJECT, Handle, vehicleModelName);
```

Comment:
We should rename this method since the native changes not only engine audio but also changes verious vehicle sounds such as horn, door, suspension, and start sequences. We could just define the signature like ForceUseAudioGameObject(string gameObjectName) if we don't have better names. Well, we can indicate that the method **GENRARLLY** accepts vehicle model names, since the same name hash should be used unless the vehicles.meta explicitly specifies the value.

I searched for occurrences for this native, and I found that the string can take any vehicle item names defined in `game.dat` files (the exe searches by `audio:/config/game.dat`). All of the 3 occurrences in ysc scripts for b2944, "BJXL_ARMENIAN_3" and "STRETCH_MICHAEL_4" is not vehicle model names ("CARBONRS" is a vehicle model name however). If you want to test this, search some `game.dat[some number].rel` for `Item type="Vehicle"` entries in CodeWalker and use names of the found entries (the game sets the hashed value on `audVehicleAudioEntity` to request a different audio, and reset to zero soon).

The doc of `FORCE_USE_AUDIO_GAME_OBJECT`:
```
//INFO: 
//PARAM NOTES:
//PURPOSE: Force this vehicle to use a particular audio game object
NATIVE PROC FORCE_USE_AUDIO_GAME_OBJECT(VEHICLE_INDEX VehicleIndex, STRING GameObjectName) = "0x1be725508ba268ef"
```